python/cudf_polars/tests/expressions/test_struct.py

```diff
@@ -17,6 +17,49 @@ def ldf():
     return pl.LazyFrame({"a": [{"b": "c", "d": "e"}, {"b": None, "d": "g"}]})
 
 
+def test_field_getitem(ldf):
+    query = ldf.select(pl.col("a").struct[0])
+    assert_gpu_result_equal(query)
+
+
+def test_field(ldf):
+    query = ldf.select(pl.col("a").struct.field("b"))
```

Comment:
I think that [`.struct.field`](https://docs.pola.rs/api/python/dev/reference/expressions/api/polars.Expr.struct.field.html#polars.Expr.struct.field) supports selecting multiple fields with `.struct.field("b", "c', ...)`. Can you add a test for that?

Also I'm seeing there are some "special" names like wildcards (`.struct.field("*")`) and regexes (`.struct.field("^a.*|b.*$")`. Can you check whether we support those and add tests if we do? (and maybe handle fallback stuff if we don't yet support them).